OlinkAnalyze/vignettes/OutlierExclusion.Rmd

```diff
@@ -0,0 +1,287 @@
+
+Similar to what was shown in the NPX distribution plot example, the QC plot can also be used to see changes in specific groups. In the example below, the samples are colored by site and Site D appears to have samples with higher sample median. 
+
+```{r}
+group_data |> 
+  olink_qc_plot(color_g = "Site")
+```
+
+
+### Labeling outliers 
+
+
+To change the threshold and outliers that are labeled, we can use `median_outlierDef`, `IQR_outlierDef`, `outlierLines`, and `label_outliers`. These arguments will plot lines at a number of standard deviations from the mean of the sample median or IQR and label the samples outside of these lines. The values of `median_outlierDef` and `IQR_outlierDef` default to 3 and may require some manual adjustment to determine the correct number to highlight the outliers.
+
+```{r}
+outlier_data |> 
+  olink_qc_plot(median_outlierDef = 2, IQR_outlierDef = 4, 
+                 outlierLines = TRUE, label_outliers = TRUE) 
+```
+
+To remove the lines and just keep the outliers labelled `outlierLines` can be set to False.
+
+```{r}
+outlier_data |> 
+  olink_qc_plot(median_outlierDef = 2, IQR_outlierDef = 4, 
+                 outlierLines = FALSE, label_outliers = TRUE) 
+```
+
+
+Once the correct outliers have been highlighted in the graph, we can then programmatically extract the outlier SampleIDs using *dplyr*.
+
+```{r}
+outliers_qc_labeled <- outlier_data |> 
+  olink_qc_plot(median_outlierDef = 2, IQR_outlierDef = 4, 
+                 outlierLines = FALSE, label_outliers = TRUE) 
+
+outliers_qc_labeled$data |> 
+  dplyr::filter(Outlier == TRUE) |> 
+  dplyr::select(SampleID) |> 
+  dplyr::distinct()
+
+```
+
+## Interpreting and Handling Outliers
+
+### When should outliers be removed
+After reviewing these plots, a list of potential outliers may be generated. Whether or not these outliers should be excluded depends on how different from the other samples the potential outliers appear and if the outliers can be traced to a biological, sample specific, or technical issue. 
+
+In the case of a biological issue, it may be useful to keep the potential outlier within the study as the sample has biological meaning, however additional normalization may be needed. For example, in a diabetes study, patients with severe diabetes may have higher protein in their urine as compared to control subject. In this case it could be useful to normalize based on total protein concentration or median NPX to set all values within the context of the study. 
+Another example is in the case of multiple batches, one batch may appear to cluster differently from another batch. In this case additional normalization such as bridging is needed to bridge the studies to the same scale.
```

Comment:
Probably also add how to handle Outliers from different panels, only the data points from those panels could be removed.